src/utils/polygon.cpp

```diff
@@ -105,21 +106,24 @@ void Polygons::makeConvex()
 {
     for (PolygonRef poly : *this)
     {
+        Polygon convexified;
         Point a = poly.back();
-        for (int i = 0; i < poly.size();)
+        for (size_t i = 0; i < poly.size(); ++i)
         {
-            Point b = poly[i];
-            Point c = poly[(i + 1) % poly.size()];
-            if (LinearAlg2D::pointIsLeftOfLine(b, a, c) >= 0)
+            const Point& b = poly[i];
+            const Point& c = poly[(i + 1) % poly.size()];
+            if (LinearAlg2D::pointIsLeftOfLine(b, a, c) < 0)
             {
                 poly.remove(i);
             }
             else
             {
+                convexified.path->push_back(b);
                 a = b;
                 i++;
```

Comment:
Should this still happen?

Reply:
Oh, maybe I made a mistake with the commit here. The entire case should be turned around (since the `if` condition is turned around). I thought I'd deleted the `poly.remove(i)` as well, but maybe I stashed stuff in between or something. This is definitely not correct.